lix/source-code/client/src/isomorphic-git-forks/basename.ts

```diff
@@ -0,0 +1,7 @@
+export function basename(path: string) {
```

Comment:
// forked files:
// link to whre from incl version, why forked, what was patched